doc/guides/backporting-to-release-lines.md

```diff
@@ -0,0 +1,124 @@
+
+```shell
+# Assuming your fork of Node.js is checked out in $NODE_DIR,
+# the origin remote points to your fork, and the upstream remote points
+# to git://github.com/nodejs/node
+cd $NODE_DIR
+git checkout v7.x-staging
+git pull -r upstream v7.x-staging
+# We want to backport pr #10157
+git checkout -b backport-10157-to-v7.x
+```
+
+* After creating the branch, apply the changes to the branch. The cherry-pick
+  will likely fail due to conflicts. In that case, you will see something this:
+
+```shell
+# Say the $SHA is 773cdc31ef
+$ git cherry-pick $SHA # Use your commit hash
+error: could not apply 773cdc3... <commit title>
+hint: after resolving the conflicts, mark the corrected paths
+hint: with 'git add <paths>' or 'git rm <paths>'
+hint: and commit the result with 'git commit'
+```
+
+* Make the required changes to remove the conflicts, and then commit the
+  changes. That can be done with `git commit`.
+* The commit message should be as close as possible to the commit message on the
+  master branch, unless the commit has to be different due to dependencies that
+  are not present in the targeted release line. The only exception is that the
+  metadata from the original commit should be removed. If a backport is
+  required, it should go through the same review steps as a commit landing
+  in the master branch.
+* Push the changes to your fork and open a pull request.
+* Be sure to target the `v7.x-staging` branch in the pull request.
+* When landing a backport commit, please include the PR-URL from the original
+  pull request as `Backport-of: <origin PR-URL>`. The `Backport-of` line should
+  be directly after the `PR-URL` line in the metadata.
+
+Below are examples of an original commit message and a backport commit message.
+
+In this example, https://github.com/nodejs/node/pull/1234 is the original pull
+request and https://github.com/nodejs/node/pull/5678 is the backport.
+
+Original:
+
+```
+lib: make something faster
+
+Switch to using String#repeat to improve performance.
+
+PR-URL: https://github.com/nodejs/node/pull/1234
+```
+
+Backport:
+
+```
+lib: make something faster
+
+Switch to using String#repeat to improve performance.
+
+PR-URL: https://github.com/nodejs/node/pull/5678
+Backport-of: https://github.com/nodejs/node/pull/1234
+```
+
+### Helpful Hints
+
+* Please include the backport target in the pull request title in the following
+  format: `(v7.x backport) <commit title>`
+  * Ex. `(v4.x backport) process: improve performance of nextTick`
+* Please include the text `Backport of #<pull request number>` in the
+  pull request description. This will link to the original pull request.
+
+In the event the backport pull request is different than the original,
+the backport pull request should be reviewed the same way a new pull request
+is reviewed. When each commit is landed, the new reviewers and the new PR-URL
+should be used.
+
+[`LTS Plan`]: https://github.com/nodejs/LTS#lts-plan
```

Comment:
Is there a reason you used 

```
[`LTS Plan`]
```

rather than:

```
[LTS Plan]
```

? It looks a bit odd to me.